docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA_RU.liquid

```diff
@@ -1,29 +1,57 @@
 {%- include getting_started/global/partials/NOTICES.liquid %}
 
+{% if page.platform_type == "existing" %}
+## Прежде чем вы начнете
+
+Если вы развернули **новый** кластер и хотите поставить в него Deckhouse, то рекомендуем вернуться на предыдущий шаг и использовать руководство по развертыванию кластера с помощью самого Deckhouse.
+
+Кластеры Kubernetes предлагаемые различными облачными провайдерами имеют свои особенности и различия, так же как и кластеры, развернутые вручную. Правильная стратегия установки Deckhouse в таких кластерах — установить Deckhouse в минимальной конфигурации, после чего включить необходимые модули.
+
+Данное руководство предлагает установку Deckhouse в **минимальной** конфигурации, с включенным модулем [внутренней документации](../../documentation/v1/modules/810-deckhouse-web/) (для примера работы web-приложения) и модулем [cert-manager](../../documentation/v1/modules/101-cert-manager/).
+
+После выполнения всех шагов данного руководства вам нужно будет самостоятельно включить интересующие вас модули. Воспользуйтесь [документацией](/documentation/), чтобы получить об этом необходимую информацию. Также, при возникновении вопросов вы можете попросить помощи [сообщества](https://deckhouse.io/ru/community/about.html).
+
+{% endif %}
+
 ## Процесс установки
 
 Вам потребуются:
 1. **Персональный компьютер**.
  
-   Компьютер, с которого будет производиться установка. Он нужен только для запуска инсталлятора Deckhouse, и не будет частью кластера. 
+   Компьютер, с которого будет производиться установка. {% unless page.platform_type == "existing" %}Он нужен только для запуска инсталлятора Deckhouse, и не будет частью кластера.{% endunless %}
    
    Требования:
    - HTTPS-доступ до container registry `registry.deckhouse.io` (установка также возможна и в [закрытом контуре](/ru/documentation/v1/deckhouse-faq.html#как-установить-deckhouse-из-стороннего-registry));
-   - SSH-доступ по ключу до узла, который будет **master-узлом** будущего кластера. 
-2. **Master-узел**.
+{%- if page.platform_type == "existing" %}
+   - Конфигурация kubectl для доступа к существующему кластеру.
+{%- else %}
+   - SSH-доступ по ключу до узла, который будет **master-узлом** будущего кластера.
+{%- endif %}
+{% if page.platform_type == "existing" %}
+1. **Существующий кластер Kubernetes**.
+
+   У вас уже должен быть работающий кластер Kubernetes, подходящий под условия [поддерживаемых версий Kubernetes и ОС](../../documentation/v1/supported_versions.html).
```

Comment:
We need add information about support k8s versions.

For example, 1.23 version is available, but we do not test Deckhouse with 1.23 version.

Reply:
Supported versions can be found [here](https://github.com/deckhouse/deckhouse/blob/8134aa29b1977b996f26e14aa947ccb2cb61dba7/candi/openapi/cluster_configuration.yaml#L71)

Reply:
I've found. Close it